docs/source/examples/Using NEOS package.ipynb

```diff
@@ -0,0 +1,1777 @@
2m2WsN7v1/NQFQ+whhEwPFOZ/dzqJWQ/k550nQVhowCNRC\nI+QRMMbcoK1iPRlAZUP9tHtwzksBCBljzZNaHYApAOYyxuY85DEdABRxzut/f+SEGB5KaIT8Towx\ncwD7AbzKOU8G8Hdo76e15SdoW2NNOOdV0FbCXsEYm/QQhx4F4IeHj5gQw0b10AjpAoyxPgBWcM5f\neAz72g/gdc75zUePjBDDQS00QroA5/w3AGcYY8JH2U/DiMkDlMwIaY1aaIQQQgwCtdAIIYQYhDaH\n7cfFxTkbGRltBRAMSnqEEEK6Dw2ABJVK9VLfvn0Lmi9oM6EZGRltdXV1DXRycioVCATUJ0kIIaRb\n0Gg0rLCwUJaXl7cVwD2jg9trfQU7OTlVUDIjhBDSnQgEAu7k5FQObQ/ivcva34aSGSGEkO6nIT+1\nyl90f4wQQohBoIRG9ApjrO8f//hHj8bXa9eudVm5cqW7LmN6VEeOHLEaNWqUX1cfd+XKle5r1651\n6erjEtJZKKERvSISifjRo0ftcnNzaWJtQsg9KKERvSIUCvncuXML/+///q9Vy2LPnj02oaGhPQMD\nA2WDBw+W3rlzxwjQtkSmTp3qPWTIEH+xWByyfft225dfftlDKpXKhg0b5q9UKhkAnDt3zrx///4B\nQUFBgUOHDvXPzMw0BoD33nvP2dfXN0gqlcomTJjQo+VxU1NTRX379g2QyWSBMpks8MSJExaAtuUV\nHh4eEBER0cPHxydo0qRJPhqNBgAQExNj7ePjE9S3b9+AmJgY27bOVaVSYfHixR7BwcGBUqlU9sEH\nHzg27rdfv34B48aN8/X19Q2aM2eOl1qtrVizZcsWe6lUKvP39w9asmSJuHFfMTEx1jKZLDAgIEA2\naNAgaeP7ycnJZuHh4QEeHh4h7733njMAVFRUCEaOHOkXEBAg8/f3D/riiy/sft9vi5CuRQmN6J3X\nXnutYP/+/fbFxcX3TCM1bty4ymvXrqUkJycnTZ8+veTdd991bVyWmZlpcvr06fSYmJj0l19+2Wf0\n6NEVN2/eTDI1NdV88803Nkqlki1btszr4MGDGYmJicnz5s0rWrVqlRgAPvroI9eEhISkmzdvJm3b\nti2zZTzu7u6qc+fO3UxKSkret2/frRUrVng1LktOTjb75JNP7qSnpydmZWWZnDhxwrK6uppFRUV5\nHzp0KP3y5cupBQUFxm2d54cffuhoY2OjTkhISL5+/Xry9u3bnVJSUkQAEB8fb7Fp06Y7qampiXK5\n3GTHjh12crnc+O233xb/97//vZmUlJR49epVi507d9revXvXKCoqynv//v0ZqampSQcOHMhoPEZ6\nerrp2bNnb16+fDn5X//6l7tSqWT79++3dnV1rU9NTU1KS0tLnDp1asWj/9YI6XzUbUP0jr29veb5\n558vXrdunbOZmZmm8f3bt2+LnnvuOY/CwkLjuro6gaenZ1NtsbFjx5abmJjw8PDwGrVazaZPn14B\nAEFBQTW3b98W3bhxwyQtLc1s9OjRUgDQaDRwcnKqB4CAgICaKVOm+EyaNKksMjKyZaFO1NXVsYUL\nF0qSkpLMBAIBMjMzTRqXhYSEVPn6+tY3HKs6IyNDZGVlpfbw8FCGhIQoASAyMrJ469atTi33e/Lk\nSeuUlBTzQ4cO2QGAQqEQJiUlmYpEIh4SElIlk8nqAGDGjBkl586dszQ2NuYDBw5UuLu7qwBg5syZ\nJWfPnrUUCoU8PDxc0bNnzzoAcHFxaSpA+tRTT5WZmZlxMzMzlb29fX12drZRnz59atasWeO5ZMkS\n8eTJk8sjIiIqH+X3RUhXoRYa0UtvvPFG/p49exyrqqqa/oajoqK8li5dWnDz5s2kzZs3ZyqVyqZl\nJiYmHACEQiGMjIy4QKBdJBAIoFKpGOec+fn51aSkpCSlpKQk3bx5M+mXX35JA4AzZ86k/elPfyqM\ni4uzCAsLk9XX31uG7P3333dxdnauT05OToqPj0+qr69vddzGY6tUKtbRc+Scsw0bNmQ1xpSTkxPf\n2Fpi7N7dtHzdUW3FFxoaqvztt9+SQkJCatasWSNetWqV2+/aOSFdjBIa0UsuLi7qiRMnlu7Zs8ex\n8T2FQiH08vKqB4Bt27Y5PMz+QkNDa0tKSoxOnjxpAQBKpZJduXLFVK1WIyMjQzRx4kTFp59+mq1Q\nKITl5eX3dHWWl5cL3dzc6oVCIT799FOHxvtZ7enVq1dtTk6OKDEx0QQAoqOj7dtab9y4ceWfffaZ\nU+M9vhs3bphUVFQIAG2XY0pKikitViMmJsZ+2LBhimHDhlXFxsZa5ebmGqlUKnz77bf2I0eOrBw5\ncmTVpUuXrBq7K/Pz8+87479cLje2srLSLF26tGT58uX5165dM+/gj5EQnaIuR6K31qxZk7d9+3an\nZq/vzp4929fFxaWuX79+VVlZWSb32745U1NTHh0dnbFs2TIvhUIhVKvVbMmSJfkhISHKOXPm+CgU\nCiHnnC1evDjf0dHxnoy1fPnygmnTpvkeOHDAbujQoYrm3aBtMTc35x9//HHmhAkT/Ozt7VUDBgyo\nTE5ONmu53ooVK4rkcrlJSEhIIOec2dvb1x89ejQDAHr16lX56quveqSkpJgNGDBA8cILL5QJhUK8\n9dZbOSNGjJByztnYsWPL/vCHP5QBwEcffSSfMmWKn0ajgYODQ/2FCxfS2osvLi7O7I033vAQCAQw\nMjLin376aav7hoR0R22Wj7l+/bo8LCysSAfxEEIe4MiRI1YbNmxwOXPmTLquYyFEV65fv+4YFhbm\n3fw96nIkhBBiECihEaJnJkyYoKDWGSGtUUIjhBBiECihEUIIMQiU0AghhBgESmiEEEIMQrdMaOnp\n6cYDBgyQ+vr6Bvn5+QX9/e9/d25clp+fLxw8eLC/RCIJHjx4sH9hYaEQAK5evWraq1
```

Comment:
I wonder if we should take integers instead of strings here... what do you think?